Cargo.toml

```diff
@@ -48,6 +48,9 @@ esp-idf-hal = { version = "0.43", default-features = false }
 embassy-time-driver = { version = "0.1", optional = true, features = ["tick-hz-1_000_000"] }
 embassy-futures = "0.1"
 
+[patch.crates-io]
```

Comment:
I've merged the PR in `embedded-svc` now. Would you mind changing this to `https://github.com/esp-rs/embedded-svc.git`?